store/src/java/com/zimbra/cs/service/account/Auth.java

```diff
@@ -446,24 +485,9 @@ private Element doResponse(Element request, AuthToken at, ZimbraSoapContext zsc,
         String skin = SkinUtil.chooseSkin(acct, requestedSkin);
         ZimbraLog.webclient.debug("chooseSkin() returned "+skin );
         if (skin != null) {
-            response.addElement(AccountConstants.E_SKIN).setText(skin);
+            response.addUniqueElement(AccountConstants.E_SKIN).setText(skin);
```

Comment:
Please change it to `addNonUniqueElement`.  That is the non-deprecated method call which does exaclty the same thing as `addElement` does - as mentioned in the JavaDoc for `addElement` explaining why it was deprecated.